src/pages/instances/forms/NetworkForm.tsx

```diff
@@ -75,6 +76,7 @@ const NetworkForm: FC<Props> = ({ formik, project, children }) => {
                   <Input
                     label="Network name"
                     name={`devices.${index}.name`}
+                    id={`networkName-${index}`}
```

Comment:
With or without the `-` before the index?
(I'd say without since it's camelCased, anyway it's better to stick with one convention and keep it everywhere)